reference/import_export/product-export.rst

```diff
@@ -40,67 +34,46 @@ We can also count a single step named ``export``.
 Product Export Step
 -------------------
 
+Due to the huge refactor of Imports and Exports steps are now services. They are injected in the export job.
```

Comment:
Due to the huge refactor of Imports and Exports, steps are now services or even:
"Steps are now symfony services injected into the export job."